skt/reporter.py

```diff
@@ -12,15 +12,16 @@
 # along with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 from __future__ import print_function
-from email.mime.application import MIMEApplication
-from email.mime.multipart import MIMEMultipart
-from email.mime.text import MIMEText
+
+import StringIO
 import gzip
 import logging
 import os
 import re
 import smtplib
-import StringIO
+from email.mime.application import MIMEApplication
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
```

Comment:
PEP8 doesn't say anything about the actual order in the groupings, but lexical order is usually used because of readability and orientation reasons. This breaks the lexical order while actually not fixing anything and making the orientation around imports more confusing?